specs/consensus/quint/driver.qnt

```diff
@@ -710,9 +707,6 @@ module driver {
             val cert = certificatesForCurrentHeight.fold (Set(), (sum, y) => y)
             val newstate = { ...state, incomingCertificates: state.incomingCertificates.exclude(Set(cert))}
             (newstate, CertificateDInput(cert))
-            
-        else if (command == TimeoutCmd and state.existsTimeout())
```

Comment:
Not sure if we should remove this. The `nextActionCommand` is only used by a test unit (`tests/state-machine/statemachineTest.qnt`) and in the case the `TimeoutCmd` is explicitly provided.